config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-periodics.yaml

```diff
@@ -0,0 +1,114 @@
+periodics:
+  - name: periodic-cluster-api-provider-vsphere-e2e-main
+    labels:
+      preset-dind-enabled: "true"
+      preset-cluster-api-provider-vsphere-e2e-config: "true"
+      preset-cluster-api-provider-vsphere-gcs-creds: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    interval: 12h
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: cluster-api-provider-vsphere
+        base_ref: master
+        path_alias: sigs.k8s.io/cluster-api-provider-vsphere
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220124-681f3531ec-1.21
```

Comment:
Was just updated to the 1.23 version, use the same. Rebase the branch and you should see the latest image tag.